simulator/virtual_machine.go

```diff
@@ -142,6 +143,29 @@ func (vm *VirtualMachine) apply(spec *types.VirtualMachineConfigSpec) {
 		}
 	}
 
+	apply2 := []struct {
```

Comment:
Can we keep this typed and avoid reflection?  For example:
```go
	applyb := []struct {
		src *bool
		dst *bool
	}{
		{spec.NestedHVEnabled, vm.Config.NestedHVEnabled},
	}

	for _, f := range applyb {
		if f.src != nil {
			f.dst = types.NewBool(*f.src)
		}
	}
```

Reply:
Sure.. Most of them are `bool`s so it's mostly what it is.